src/bootstrap/compile.rs

```diff
@@ -294,9 +298,7 @@ fn compiler_file(compiler: &Path, file: &str) -> PathBuf {
 }
 
 pub fn create_sysroot(build: &Build, compiler: &Compiler) {
-    let sysroot = build.sysroot(compiler);
-    let _ = fs::remove_dir_all(&sysroot);
```

Comment:
Ok, it looks like all the remaining sysroot components are already built somewhere else and then copied to sysroot directory.